pkg/scheduler/cache/cache.go

```diff
@@ -270,30 +270,35 @@ func (cache *schedulerCache) getPodResizeRequirements(pod *v1.Pod, resizeRequest
 	return resizeContainersMap, podResource, nil
 }
 
-func restoreResources(pod *v1.Pod, restoreResources string) error {
+func (cache *schedulerCache) restoreResources(oldPod, newPod *v1.Pod, restoreResources string) error {
```

Comment:
this needs a unit test

Reply:
i have a work item tracking the task of adding more unit test coverage. will come in a subsequent commit.